src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,18 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += f'\nAuthorization for identifier {authzr.body.identifier} failed.'
+                       msg += '\nHere are the challenges that were not fulfilled:'
+                       for challenge in authzr.body.challenges:
+                         msg += \
+                               f'\nChallenge Type: {challenge.chall.typ}' \
+                               f'\n\nError information:' \
+                               f'\n- Type: {challenge.error.typ if challenge.error else ""}' \
+                               f'\n- Details: {challenge.error.detail if challenge.error else ""}\n\n'
```

Comment:
@william-gr We might also do something with those else portions...
Maybe something like:

```suggestion
                               f'\n- Type: {challenge.error.typ if challenge.error else "No error type found"}' \
                               f'\n- Details: {challenge.error.detail if challenge.error else "No error details found"}\n\n'
```